Index: kdewebkit-4.98.0/CMakeLists.txt
===================================================================
--- kdewebkit-4.98.0.orig/CMakeLists.txt	2014-03-28 18:16:37.000000000 +0000
+++ kdewebkit-4.98.0/CMakeLists.txt	2014-04-02 13:13:20.014736722 +0100
@@ -6,7 +6,7 @@
 
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR})
 
-set(REQUIRED_QT_VERSION "5.2")
+set(REQUIRED_QT_VERSION "5.1")
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Core Widgets WebKitWidgets Network)
 include(KDEInstallDirs)
 include(KDEFrameworkCompilerSettings)
